contents/dataframesmeta_transform.md

```diff
@@ -0,0 +1,87 @@
+## Column Transformation {#sec:dataframesmeta_transform}
+
+Whereas the `@select` macro variants performs column selection,
+the `@transform` macro variants do _not_ perform any column selection.
+It can either _overwrite_ existent columns or _create_ new columns that will be added to the right of our `DataFrame`.
+
+For example, the previous operation on `:grade` can be invoked as a transformation with:
+
+```jl
+sco("""
+# rtransform 1 # hide
+@rtransform df :grade_100 = :grade * 10
+"""; process=without_caption_label
+)
+```
+
+As you can see, `@transform` does not perform column selection,
+and the `:grade_100` column is created as a new column and added to the right of our `DataFrame`.
+
+DFM macros also support `begin ... end` statements.
```

Comment:
Didn't we have this sentence before or is this about something else?

Reply:
No this is the first one. I will fix the following ones accordingly